Bucketeer/Sources/Internal/Model/MetricsEventData.swift

```diff
@@ -5,37 +5,114 @@ enum MetricsEventData: Hashable {
     case getEvaluationSize(GetEvaluationSize)
     case timeoutError(TimeoutError)
     case networkError(NetworkError)
+    case badRequestError(BadRequestError)
+    case unauthorizedError(UnauthorizedError)
+    case forbiddenError(ForbiddenError)
     case internalSdkError(InternalSdkError)
+    case internalServerError(InternalServerError)
+    case unknownError(UnknownError)
 
     struct GetEvaluationLatency: Codable, Hashable {
         let apiId: ApiId
         let labels: [String: String]
         let duration: Duration
         var protobufType: String? = "type.googleapis.com/bucketeer.event.client.GetEvaluationLatencyMetricsEvent"
+
+        func hash(into hasher: inout Hasher) {
+            hasher.combine(apiId)
+            hasher.combine(duration)
+        }
     }
 
     struct GetEvaluationSize: Codable, Hashable {
         let apiId: ApiId
         let labels: [String: String]
         let size_byte: Int64
         var protobufType: String? = "type.googleapis.com/bucketeer.event.client.GetEvaluationSizeMetricsEvent"
```

Comment:
The Size metrics event is old.
Please use the `SizeMetricsEvent` and change the struct's name.

```swift
type.googleapis.com/bucketeer.event.client.SizeMetricsEvent
```

Reply:
Updated https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/cde633891b0ccb15c434164160feec2bbe3da073#diff-21e1c84fd4bacc9267f4a4cd72b9f6a4a37dd6ab724b13ee0d18973da4b186b9L31-R34